### DISCUSSION TITLE
Error when publishing stream from OBS Studio

### DISCUSSION THREAD

**Discussion by timofeys1** [bluenviron/mediamtx#300] [Questions]
## Which version are you using?

v0.15.0

## Which operating system are you using?

OS

- [x] Windows

Architecture

- [x] amd64

## Describe the issue

When trying to publish a stream from OBS Studio 26.1.1 to the path `stream` I get an error:

```
ERR: invalid path name: can't begin with a slash (/stream)
```

I managed to get around the error by disabling the check and reading from `/stream` instead.
https://github.com/aler9/rtsp-simple-server/blob/22ab4d489e64213bf140e3f32de8bc844cf325cc/internal/conf/path.go#L52-L54

![ScreenShot](https://user-images.githubusercontent.com/8717108/111511518-e21b1280-875f-11eb-811f-a447b4631c19.JPG)

It seems that OBS Studio prepends an unexpected slash in my setup.

## Describe how to replicate the issue

1. Configure OBS Studio 26.1.1 to publish as follows

![OBSSetup](https://user-images.githubusercontent.com/8717108/111512614-fb708e80-8760-11eb-98be-558a6dce4154.JPG)

2. Press the "Start Streaming" button
3. rtsp-simple-server outputs an error

**Comment by aler9** [comment]
Hello, just configure OBS Studio in this way:

```
Server: rtmp://localhost/stream
Stream Key: (none)
```

**Comment by casualshammy** [comment]
Hello. Excuse me for hijacking. 
I got the same error when I was trying to understand how to setup authentication (OBS with RTMP). Is it possible?

**Comment by aler9** [comment]
Instructions on how to publish from OBS studio, with or without authentication, have been added to the readme

https://github.com/aler9/rtsp-simple-server#publish-from-obs-studio

**Comment by timofeys1** [comment]
That was silly of me. Thanks!

[a comment by an automated account is not shown]
